### ISSUE TITLE
A problem for fine_tuning test

### ISSUE THREAD

**Issue by tzczsq** [facebook/C3D#101]
Hi, I have fine_tuned your provided model "conv3d_deepnetA_sport1m_iter_1900000" on my training data. However, an error "Check failed: read_status Testing must not miss any example" exists when performing test on my testing data ( it contains 25 video files (avi) split into 426 clips, each of which has 16 frames ). I am sure that my test_list for testing data is right.
it seems that the two parameters, i.e., batch_size, and test iterations for test, are related to the number of the total 426 clips  or the total video 25 files ? How to decide these two parameters?
Experiment settings:
1)Batch_size=20, iteration =21:
E0403 10:07:27.228180  9861 test_net.cpp:45] Running 21 iterations.
E0403 10:07:27.994477  9861 test_net.cpp:51] Batch 0, accuracy: 0
E0403 10:07:28.436292  9861 test_net.cpp:51] Batch 1, accuracy: 0
E0403 10:07:28.884855  9861 test_net.cpp:51] Batch 2, accuracy: 0.2
E0403 10:07:29.336796  9861 test_net.cpp:51] Batch 3, accuracy: 0.05
E0403 10:07:29.777667  9861 test_net.cpp:51] Batch 4, accuracy: 0.2
F0403 10:07:29.982298  9907 video_data_layer.cpp:123] Check failed: read_status Testing must not miss any example
*** Check failure stack trace: ***

Batch_size=20, iteration =4:
E0403 10:11:54.028420 10501 test_net.cpp:45] Running 4 iterations.
E0403 10:11:54.655246 10501 test_net.cpp:51] Batch 0, accuracy: 0
E0403 10:11:55.092314 10501 test_net.cpp:51] Batch 1, accuracy: 0
E0403 10:11:55.559475 10501 test_net.cpp:51] Batch 2, accuracy: 0.2
E0403 10:11:55.999429 10501 test_net.cpp:51] Batch 3, accuracy: 0.05
E0403 10:11:55.999517 10501 test_net.cpp:54] Test accuracy: 0.0625

In this case (Batch_size=20, iteration =4), I get test accuracy: 0.0625, but It is too low. Is it correct?

2) Batch_size=1, iteration =134:
E0403 10:31:37.610033 13705 test_net.cpp:45] Running 134 iterations.
E0403 10:31:37.767882 13705 test_net.cpp:51] Batch 0, accuracy: 0
E0403 10:31:37.794507 13705 test_net.cpp:51] Batch 1, accuracy: 0
......===
E0403 10:31:40.819099 13705 test_net.cpp:51] Batch 128, accuracy: 0
E0403 10:31:40.842851 13705 test_net.cpp:51] Batch 129, accuracy: 0
E0403 10:31:40.866672 13705 test_net.cpp:51] Batch 130, accuracy: 0
E0403 10:31:40.890329 13705 test_net.cpp:51] Batch 131, accuracy: 0
E0403 10:31:40.913712 13705 test_net.cpp:51] Batch 132, accuracy: 0
E0403 10:31:40.937286 13705 test_net.cpp:51] Batch 133, accuracy: 0
E0403 10:31:40.937358 13705 test_net.cpp:54] Test accuracy: 0.0746269

When I use iterations=135 (more than 134), the same error emerges, why?

**Comment by dutran** [comment]
@tzczsq same as the other post by you, make sure your input clips are well read.

**Comment by beihaidao** [comment]
@dutran hi,du. 
I also has the confuse about this problen.
(1)if i set the bach_size=1 && iteration=100 ,then i will get the error result,like:
E0827 16:35:46.676556 30492 test_net.cpp:51] Batch 87, accuracy: 0
F0827 16:35:46.677166 30587 video_data_layer.cpp:123] Check failed: read_status Testing must not miss any example
*** Check failure stack trace: ***
Aborted (core dumped)
from this error result i  find that : batch_size times iteration must less than 83, in this case can i get the right result,but i dont know this is why? i known my computer calculated performance is not so good, so i reduce the batch_size from 30 to 8, but if the iteration is too small ,and the result is not convincing. right?
so , how to solve the batch_size and iterations problem?
look forward to your reply. 
thanks a lot.

**Comment by linguoting** [comment]
@beihaidao Hi,I also has the confuse about this problen.Did you solve this problem?
